elements/pf-clipboard-copy/demo/pf-clipboard-copy.html

```diff
@@ -0,0 +1,87 @@
+<link rel="stylesheet" href="demo.css">
+<script type="module" src="pf-clipboard-copy.js"></script>
+
+<section>
+  <h3>Default</h3>
+  <pf-clipboard-copy value="This is editable"></pf-clipboard-copy>
+</section>
+
+<section>
+  <h3>Read-only</h3>
+  <pf-clipboard-copy readonly value="This is read-only"></pf-clipboard-copy>
+</section>
+
+<section>
+  <h3>Expanded</h3>
+  <pf-clipboard-copy expandable>
+    Got a lot of text here, need to see all of it?
+    Click that arrow on the left side and check out the resulting expansion.
+  </pf-clipboard-copy>
+</section>
+
+<section>
+  <h3>Read-only expanded</h3>
+  <pf-clipboard-copy expandable readonly>
+    Got a lot of text here, need to see all of it?
+    Click that arrow on the left side and check out the resulting expansion.
+  </pf-clipboard-copy>
+</section>
+
+<section>
+  <h3>Read-only expanded by default</h3>
+  <pf-clipboard-copy expandable readonly expanded>
+    <p>Got a lot of text here, need to see all of it? Click that arrow on the left side and check out the resulting expansion.</p>
+    <p>asodifna osdif</p>
+  </pf-clipboard-copy>
+</section>
+
+<section>
+  <h3>JSON object (wrap code with pre)</h3>
+  <pf-clipboard-copy code expandable expanded>
```

Comment:
Heading says `(wrap code with pre)` but code is not wrapped. Instead element uses `code`. Change heading?